tests/starship/tests/go.mod

```diff
@@ -0,0 +1,173 @@
+module github.com/cosmos-sdk/tests/starship/tests
+
+go 1.20
+
+replace github.com/strangelove-ventures/lens => github.com/Anmol1696/lens v0.1.1-0.20230705212610-c00628a886a0
+
+require (
+	github.com/cosmology-tech/starship/clients/go/client v0.0.0-20230715183507-198fb2b3ee2e
+	github.com/cosmos/cosmos-sdk v0.47.3
```

Comment:
If the job is supposed to be run on every push on main, this does not make sense it uses v0.47.3